projects/go-lib/src/styles/_forms.scss

```diff
@@ -106,6 +113,66 @@ $input--disabled-background: rgba($base-dark-secondary, .4);
   }
 }
 
+/* Hide the default checkbox */
+.go-form__checkbox--hidden {
+  cursor: pointer;
+  height: 0;
+  opacity: 0;
+  position: absolute;
+  width: 0;
+}
+
+/* Create a custom checkbox */
+.go-form__custom-checkbox {
+  background-color: $base-light;
+  border: 1px solid lighten($base-light-secondary, 13.5%);
+  border-radius: 2px;
```

Comment:
I think we can push this to 3px with our `global-radius` variable. Thoughts?

Reply:
It looks noticeably more rounded than the design spec that way, but it still looks fine in my opinion. 

So I guess it comes down to how important it is that we match the design spec as closely as possible vs how much we want to keep things consistent, e.g. by using our global variables 🤷‍♂

Reply:
I would say that we should strive for consistency. Let's use the variable.